### ISSUE TITLE
Moving to RV2 Stable

### ISSUE THREAD

**Issue by thatsIch** [AppliedEnergistics/Applied-Energistics-2#1287]
Compiling a list of stuff which needs to be done before moving to release,
accepting suggestions, also feel free to discuss it, or just be done as optional, like a nice to have or which could be shifted to be done in a release-release

- [x] https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/329

Nice to have

- [x] https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/456

**Comment by Nikkolino99** [comment]
what about a ore dictionary card for buses and interfaces?

**Comment by thatsIch** [comment]
That would be a feature request, feature requests will be added in the next alpha. Its more meant for cirtical bugs and enhancements, which should be in a release version. Minor bugs and enhancements can be added through out the full release time.

**Comment by Nikkolino99** [comment]
OK! That's Hood, hope that this will ne implemented
Il 19/apr/2015 16:36, "thatsIch" <notifications@github.com> ha scritto:

**Comment by yueh** [comment]
Also depending on what this should actually be, it is either already available with fuzzy cards or a existing feature request. (deleting these comments is probably an idea to keep the issue clean)

**Comment by thatsIch** [comment]
Added https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/329 and https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/456 by @yueh's suggestion

**Comment by TheJulianJES** [comment]
@thatsIch I guess you mean have and not habe?
